src/Database/Attach/File.php

```diff
@@ -786,10 +794,9 @@ protected function putFile($sourcePath, $destinationFileName = null)
          */
         if (
             !FileHelper::isDirectory($destinationPath) &&
-            !FileHelper::makeDirectory($destinationPath, 0777, true, true) &&
-            !FileHelper::isDirectory($destinationPath)
+            !FileHelper::makeDirectory($destinationPath, 0777, true, true)
```

Comment:
What was the reason for this change?

Reply:
The second check to `isDirectory` is useless. If `makeDirectory` fails, it's already returning `false`.